builder/image/Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM ubuntu-debootstrap:14.04
 ENV DEBIAN_FRONTEND noninteractive
 
 # install common packages
-RUN apt-get update && apt-get install -y curl net-tools sudo
+RUN apt-get update && apt-get install -y curl net-tools
```

Comment:
let's keep `sudo` in here. We [do this intentionally](https://github.com/deis/deis/blob/master/controller/Dockerfile#L6) in other components such that this layer is shared across all components.